test/integration/pages/home.py

```diff
@@ -0,0 +1,78 @@
+"""Representation of the Home page for lockbox."""
+
+from pypom import Page, Region
+from selenium.webdriver.common.by import By
+
+
+class Home(Page):
+    """Contain the locators and actions relating to the home page."""
+
+    _entries_locator = (By.CSS_SELECTOR,
+                        'ul li div.item-summary__item-summary___2v2KL')
```

Comment:
Can you use the CSS class name munger function I wrote here? That would make this easier for people in the future to understand. Likewise with the other munged class names.

Reply:
@jrbenny35: The latest Python 3.x code and example can be found in https://github.com/mozilla-lockbox/lockbox-extension/issues/52#issuecomment-342677964

Reply:
I'm curious to see how we'll do this within selectors that contain more than just a single class name. I'm concerned that this will degrade readability/maintainability even further.

Reply:
I think dave brings up a good point. There are multiple ```button```'s so using the munged class name function will only solve 1 part of the selector tree.

Reply:
I'm 👍 on giving the munging function a try, within a limited scope this quarter to help us understand if it does add value as well as what artifacts it creates that are negative.

Mid-to-long term, @jimporter and the team will be maintaining these tests. Let's experiment and move forward by using @jimporter's munging function.

Reply:
Here's what it'd look like for `_entries_locator`:

```python
_entries_locator = (By.CSS_SELECTOR,
                    'ul li div.{}'.format(munged_class_name('webextension/manage/components/item-summary.css', 'item-summary')))
```

Makes for some pretty verbose readin'. Not sure if it'd be prettier to move the munge code into an intermediate variable, or if that'd just make a variable explosion. But adding a second munged selector would expand this by roughly `munged_class_name('webextension/manage/components/item-summary.css', 'item-summary')` width.